player-portal/src/styles.css

```
@@ -1,97 +1,348 @@
+  --shadow-md: 0 4px 12px rgba(15, 23, 42, 0.08), 0 2px 4px rgba(15, 23, 42, 0.04);
+  --shadow-lg: 0 12px 32px rgba(15, 23, 42, 0.14), 0 4px 10px rgba(15, 23, 42, 0.06);
+  --shadow-glow: 0 8px 28px rgba(5, 150, 105, 0.28);
+
+  /* Scale */
+  --r-sm: 8px;
+  --r-md: 12px;
+  --r-lg: 18px;
+  --r-xl: 24px;
+  --r-pill: 999px;
+
+  --font: 'Inter', system-ui, -apple-system, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)